google-cloud-spanner/src/main/java/com/google/cloud/spanner/spi/v1/SpannerRpc.java

```diff
@@ -34,10 +34,7 @@
 import com.google.protobuf.Empty;
 import com.google.protobuf.FieldMask;
 import com.google.spanner.admin.database.v1.*;
-import com.google.spanner.admin.instance.v1.CreateInstanceMetadata;
-import com.google.spanner.admin.instance.v1.Instance;
-import com.google.spanner.admin.instance.v1.InstanceConfig;
-import com.google.spanner.admin.instance.v1.UpdateInstanceMetadata;
+import com.google.spanner.admin.instance.v1.*;
```

Comment:
Google style requires us to never use wildcard imports: https://google.github.io/styleguide/javaguide.html

Reply:
Made this change.